server/config/mq-config.js

```diff
@@ -8,6 +8,20 @@ const mqSchema = joi.object({
     reconnect_Limit: joi.number().default(10),
     transport: joi.string().default('tcp')
   },
+  sqsCalculationQueue: {
+    url: joi.string().default(''),
```

Comment:
You can allow null and empty strings by telling joi that you allow them. `url: joi.string().default('').allow('')` you can also use `.optional()`